pkg/deploy/dashboard/deployment_dashboard.go

```diff
@@ -56,6 +56,12 @@ func (d *DashboardReconciler) getDashboardDeploymentSpec(ctx *deploy.DeployConte
 		corev1.EnvVar{
 			Name:  "CHE_URL",
 			Value: util.GetCheURL(ctx.CheCluster)},
+		corev1.EnvVar{
+			Name:  "CHE_NAMESPACE",
+			Value: ctx.CheCluster.Namespace},
```

Comment:
BTW, namespace can be read from file `/var/run/secrets/kubernetes.io/serviceaccount/namespace`.